packages/pn-pa-webapp/src/pages/components/NewNotification/Recipient.tsx

```diff
@@ -71,6 +72,11 @@ const Recipient = ({ onConfirm }: Props) => {
           .required('Campo obbligatorio')
           .matches(pIvaRegex, 'Il codice fiscale inserito non è corretto'),
         noticeCode: yup.string().required('Campo obbligatorio'),
+        digitalDomicile: yup.string().when('showDigitalDomicile', {
+          is: true,
+          then: yup.string().email('Indirizzo PEC non valido').required('Campo obbligatorio'),
```

Comment:
@AndreaCimini90 forse qui vanno aggiunti gli asterischi?

Reply:
Fatto in riga 325